backup/setup/school-external-tools.json

```diff
@@ -118,24 +118,5 @@
 		},
 		"schoolParameters": [],
 		"isDeactivated": false
-	},
-	{
```

Comment:
Why is this external tool being removed in this PR?

Reply:
Seed data produces errors as the linked tool is not present anymore, hence non of the remaining tools are shown. Error is only shown in console and was overlooked it seems.